crates/epaint/src/text/fonts.rs

```diff
@@ -762,13 +905,26 @@ impl GalleyCache {
                 cached.galley.clone()
             }
             std::collections::hash_map::Entry::Vacant(entry) => {
-                let galley = super::layout(fonts, job.into());
-                let galley = Arc::new(galley);
-                entry.insert(CachedGalley {
-                    last_used: self.generation,
-                    galley: galley.clone(),
-                });
-                galley
+                if job.break_on_newline {
```

Comment:
This deserves a comment

Reply:
I added *a* comment, let me know if you feel this is sufficient or if you had something else in mind.